getting-started-tutorial/tutorial-1-MLRun-basics.ipynb

```diff
@@ -0,0 +1,1890 @@
+{
```

Comment:
<p>Maybe add a small line about getting the schedule results from the run db? </p><p>also, maybe <code>pprint</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>